pkgs/by-name/ya/yaml2nix/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+  testers,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "yaml2nix";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "euank";
+    repo = "yaml2nix";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-DkHWWpvBco2yodyOk40LjTNcoaJ1bFKf0JY9OwWgy5M=";
+  };
+
+  cargoHash = "sha256-3N0V/5lqiE2lTXu9oUsO3FOXhWqkNlomceqIVDGGuOM=";
+
+  passthru = {
+    tests.convert = testers.runCommand {
+      name = "${finalAttrs.pname}-convert-test";
+      nativeBuildInputs = [ finalAttrs.finalPackage ];
+      script = ''
+        cat <<EOF > test.yaml
+        foo: "bar"
+        baz:
+          - qux
+        EOF
+
+        cat <<EOF > expected.nix
+        { foo = "bar"; baz = [ "qux" ]; }
+        EOF
+
+        yaml2nix test.yaml > test.nix
+
+        diff test.nix expected.nix && touch $out
+      '';
+    };
+    updateScript = nix-update-script { };
+  };
```

Comment:
Version check hook id the program supports it

Reply:
Can you clarify that? I checked yaml2nix source code and it doesn't print version/help

Reply:
You're good then